ONSdigital/sdx-transform-cs
# changed the title for vacancy images to current spec.

## What? and Why?
The image titles for the Monthly and Quarterly vacancies were the wrong way round, this PR corrects this.

## How to test
- Spin up sdx-compose
- Put through a 182-184 to see if the image shows Quarterly
- Put through a 185 to see if the image shows Monthly

## Checklist
